docs/knative-eventing-mesh/08-01-configure-kafka-channel.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Configure Kafka Channel
+type: Tutorials
+---
+
+Instead of the default Channel implementation, you can use the Knative-compatible [Kafka Channel](https://github.com/kyma-incubator/knative-kafka). To ensure Kafka works properly, you must:
+
+* Install the `knative-eventing-kafka`  component to deploy the Kafka controller
```

Comment:
I guess two spaces were not supposed. Here is the review suggestion:

```suggestion
* Install the `knative-eventing-kafka` component to deploy the Kafka controller
```

Reply:
👍